app/src/main/java/com/eveningoutpost/dexdrip/webservices/WebServiceTreatments.java

```diff
@@ -0,0 +1,124 @@
+package com.eveningoutpost.dexdrip.webservices;
+
+import android.util.Log;
+
+import com.eveningoutpost.dexdrip.Home;
+import com.eveningoutpost.dexdrip.Models.BgReading;
+import com.eveningoutpost.dexdrip.Models.DateUtil;
+import com.eveningoutpost.dexdrip.Models.Treatments;
+import com.eveningoutpost.dexdrip.Models.UserError;
+import com.eveningoutpost.dexdrip.UtilityModels.NanoStatus;
+import com.eveningoutpost.dexdrip.UtilityModels.Pref;
+import com.eveningoutpost.dexdrip.UtilityModels.SensorStatus;
+import com.eveningoutpost.dexdrip.dagger.Singleton;
+import com.eveningoutpost.dexdrip.utils.DexCollectionType;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import static com.eveningoutpost.dexdrip.wearintegration.ExternalStatusService.getLastStatusLine;
+import static com.eveningoutpost.dexdrip.wearintegration.ExternalStatusService.getLastStatusLineTime;
+
+
+/**
+ * Created by jwoglom
```

Comment:
Please either use valid javadoc tags (`@author`) or leave the author out. Your name is logged with the commit. See [Stop using Javadoc @author tag](https://www.vojtechruzicka.com/stop-using-javadoc-author-tag) for the background.

Reply:
Done. This was done automatically by a really old version of intellij